nautobot/users/views.py

```diff
@@ -213,6 +226,130 @@ def post(self, request):
         )
 
 
+#
+# Saved Views
+#
+
+
+class SavedViewUIViewSet(
+    ObjectDetailViewMixin,
+    ObjectBulkDestroyViewMixin,
+    ObjectChangeLogViewMixin,
+    ObjectDestroyViewMixin,
+    ObjectEditViewMixin,
+    ObjectListViewMixin,
+):
+    queryset = SavedView.objects.all()
+    filterset_class = SavedViewFilterSet
+    table_class = SavedViewTable
+    action_buttons = ("export",)
+
+    def get_table_class_string_from_view_name(self, view_name):
+        app_label, model_name = view_name.split(":")
+        model_name = model_name.replace("_list", "")
+        model = apps.get_model(app_label=app_label, model_name=model_name)
+        return f"{model.__name__}Table"
```

Comment:
can we use `get_table_for_model` here? Maybe we should add a reverse lookup `get_model_for_view_name` in `nautobot.core.utils.lookup` if we don't already have something like that?